src/main/java/frc/robot/RobotContainer.java

```diff
@@ -109,8 +124,8 @@ private void configureBindings() {
                     .onFalse(new InstantCommand(() -> intakeSubsystem.run(IntakeAction.NONE)));
         }
 
-        if (Constants.FeatureFlags.runScoring) {
-
+        if (FeatureFlags.runScoring && Robot.isReal()
```

Comment:
Why are we making this change? 

One core idea behind the feature flags is to make the configuration identical in sim as on the robot. The simulated robot is a "digital twin" to the real life robot. So if we have two sets of flags that could result in configuration management problems ("it works in sim but not on the robot" because the feature flags aren't harmonized, or worse "the subsystem was disabled in sim but not in real life").